packages/elements-core/src/components/TableOfContents/TableOfContents.tsx

```diff
@@ -261,8 +265,9 @@ const Group = React.memo<{
         depth={depth}
         isActive={showAsActive}
         icon={
-          NODE_TITLE_ICON[item.title] && (
```

Comment:
when displaying the zoom example spec, this was causing an endpoint group to have this icon - so added a type prop (not sure if dev portal's toc needs to include this or not)